test/test_environment.lua

```diff
@@ -572,8 +661,8 @@ function test_env.main()
 
    local install_env_vars = {
       LUAROCKS_CONFIG = test_env.testing_paths.testing_dir .. "/testing_config.lua",
-      LUA_PATH = "",
-      LUA_CPATH = ""
+      LUA_PATH,
```

Comment:
cmd on windows took as path empty space, so I rewritten it like this, maybe hisham knows another solution